src/tools/tidy/src/ui_tests.rs

```diff
@@ -7,7 +7,7 @@ use std::path::Path;
 
 const ENTRY_LIMIT: usize = 1000;
 // FIXME: The following limits should be reduced eventually.
-const ROOT_ENTRY_LIMIT: usize = 984;
+const ROOT_ENTRY_LIMIT: usize = 985;
```

Comment:
Shouldn't need this anymore, right?

Reply:
same as above

Reply:
Really? That's surprising - we aren't adding anything to the root test directory anymore...

Reply:
Yeah it's surprising for me as well 🤔